tests/explainers/test_linear.py

```diff
@@ -116,7 +112,7 @@ def test_perfect_colinear():
     X.iloc[:, 3] = 0 # test null features
     model = LinearRegression()
     model.fit(X, y)
-    explainer = shap.LinearExplainer(model, X, feature_dependence="correlation")
+    explainer = shap.LinearExplainer(model, X, feature_perturbation="correlation_dependent")
```

Comment:
so is this a breaking change? Seems like you just aligned the code to the docs but this might break people's code, doesn't it?

Reply:
Yep, this is a breaking change.

The code previously raised a warning when the old deprecated parameter names were used; as per the PR description this was over 4 years ago, so now seems like a fair time to go ahead and drop support for them.

Reply:
Sorry, did not realize the breaking label was there.